VM/plot_vm.py

```diff
@@ -24,32 +60,50 @@ def plot_vm(
 
     Parameters
     ----------
-    vm_params_dict :
-    srf_corners :
-    land_outline_path :
-    centre_line_path :
-    mag :
-    outdir :
-    ptemp :
-    logger :
+    vm_params_dict : dict
+        Dictionary extracted from vm_params.yaml file.
+    srf_corners : np.ndarray.
+        Corners of SRF planes. Formatted as [[[lon,lat],[lon,lat],[lon,lat],[lon,lat]],...]
+    land_outline_path : Path
+        Path to the land outline file
+    centre_line_path : Path
+        Path to the centre line file
+    mag : float
+        Magnitude of the event
+    outdir : Path
+        Output directory
+    ptemp : Path
+        Temporary directory
+    logger : Logger
+        Default is qclogging.get_basic_logger()
     """
 
+    from rel2vm_params import write_srf_path
```

Comment:
Does this need to be in the function?

Reply:
Yes, we need more significant refactoring if we want to avoid a cyclic import..